src/main/kotlin/com/cognifide/gradle/aem/instance/tail/InstanceLoggingInfo.kt

```diff
@@ -0,0 +1,12 @@
+package com.cognifide.gradle.aem.instance.tail
+
+import com.cognifide.gradle.aem.common.instance.Instance
+import java.time.ZoneId
+
+data class InstanceLoggingInfo(val name: String, val zoneId: ZoneId) {
+
+    companion object {
+        fun default() = InstanceLoggingInfo("unspecified", ZoneId.systemDefault())
```

Comment:
unspecified? why is it here? when it could be visible?

Reply:
In tests